bokehjs/src/coffee/common/plot_widget.coffee

```diff
@@ -15,7 +15,7 @@ class PlotWidget extends ContinuumView
 
   bind_bokeh_events: () ->
 
-  request_render: () ->
+  request_render: () =>
```

Comment:
I have mixed feeling about this change. I think it would be more robust to leave `request_render()` as-is and in the later code use `() => @renderer.request_render()`. I know it adds one level of indirection, but this isn't speed critical code.